cmd/mycelo/genesis_commands.go

```diff
@@ -99,7 +99,7 @@ var createGenesisFromConfigCommand = cli.Command{
 func readBuildPath(ctx *cli.Context) (string, error) {
 	buildpath := ctx.String(buildpathFlag.Name)
 	if buildpath == "" {
-		buildpath = path.Join(os.Getenv("CELO_MONOREPO"), "packages/protocol/build/contracts")
+		buildpath = path.Join(os.Getenv("CELO_MONOREPO"), "packages/protocol/build")
```

Comment:
is this ok?

Reply:
Yes, the monorepo changed their build folder to have `contracts`, and `contracts-mento`. As they have some contracts that collide with their names, I preferred to change our mycelo implementation to decide which folder to use